Frameworks/jdi-core/src/main/java/com/epam/jdi/uitests/core/aop/PrintProcessor.java

```diff
@@ -37,7 +53,7 @@ public Object action(ProceedingJoinPoint joinPoint) {
                 : printAnnotation.value();
         if (joinPoint.getArgs().length > 0)
             actionName = MessageFormat.format(actionName, joinPoint.getArgs());
-        return action.execute(actionName, joinPoint);
```

Comment:
What about that?

Reply:
SonarLint adviced me to rename the "action" class field